packages/mui-material/src/Select/SelectInput.js

```diff
@@ -158,6 +153,23 @@ const SelectInput = React.forwardRef(function SelectInput(props, ref) {
   const { current: isOpenControlled } = React.useRef(openProp != null);
   const [menuMinWidthState, setMenuMinWidthState] = React.useState();
 
+  const open = displayNode !== null && openState;
+
+  const ownerState = {
+    ...props,
+    variant,
+    value,
+    open,
+    error,
+  };
+
+  const paperProps = {
+    ...MenuProps.PaperProps,
```

Comment:
We do have `mergeSlotProps` utility, can you check if it's possible to use that here

Reply:
I think it's the same reason as https://github.com/mui/material-ui/pull/44913/files#r1957728696, i.e to use the `ownerState` of Select not the Menu.